waspc/test/Generator/WebAppGenerator/EntityGenerator/EntityFormGeneratorTest.hs

```diff
@@ -1,4 +1,4 @@
-module Generator.Entity.EntityFormTest where
+module Generator.WebAppGenerator.EntityGenerator.EntityFormGeneratorTest where
```

Comment:
We are now putting everywhere "generator" I see, is that also some convention?

Reply:
Ah yes!
I was thinking that it is not obvious is some module is Generator (containing generator functions) or it is something more general, like FileDraft and similar.
So, I concluded that we could just prefix all the Generator modules with Generator!
I think now it is more obvious who is doing what, but what i don't like, is that now names are so long :(.
I spent long time renaming this unfortunately, so that also slows me down with wanting to try to experiment more.
I am thinking we could leave it and see how it goes. ALtnerative could be to use Gen or just G instead of Generator.
Or, we could drop it completely.
What do you think?